### DISCUSSION TITLE
benchmarks makefile needs update?

### DISCUSSION THREAD

**Discussion by hst-m** [uNetworking/uWebSockets#1145] [Q&A] [answered]
getting make error in uWebSockets/benchmarks looks like need to update makefile for new uSockets sni_tree.cpp file `undefined reference to 'sni_find'`

**Comment by ghost** [comment]
crypto/sni_tree.cpp is needed

**Comment by hst-m** [comment]
I added that file to /benchmarks Makefile but is throwing errors, I think clang is for c not c++ https://github.com/uNetworking/uWebSockets/blob/master/benchmarks/Makefile

`error: no member named 'string_view' in namespace 'std'`
`error: use of undeclared identifier 'children'`

**Comment by sab24** [comment, marked as answer]
I added a pull request #1156 that should solve the compilation issues.

**Reply by hst-m** [reply]
this worked for me thanks
